lms/djangoapps/philu_api/helpers.py

```diff
@@ -59,6 +63,15 @@ def get_social_sharing_urls(course_url, meta_tags, tweet_text=None):
 
 
 def _compile_social_sharing_url(share_url, course_url, url_param, utm_source, text=None):
+    """
+    Create social sharing url by combining url and parameters
+    @param share_url: social platform url
+    @param course_url: course url
+    @param url_param: url parameters
+    @param utm_source: social platform source i.e. Facebook, LinkedIn, Twitter
+    @param text: url text parameter
+    @return: social sharing url
+    """
```

Comment:
Aren't we following Google Style docstrings? They are like:

    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

https://arbisoft.slack.com/archives/CGYKGH4P5/p1597833417031500

Reply:
As discussed, this style is recommended not enforced. edX doesn't enforce this style in their code base as well.